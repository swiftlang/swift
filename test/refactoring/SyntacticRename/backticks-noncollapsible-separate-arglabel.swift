struct Foo {
    /*test:def*/subscript(`x` y: Int) -> Int { fatalError() }
}
Foo()/*test:ref*/[`x`: 10]

// REQUIRES: swift_swift_parser
// RUN: %empty-directory(%t.ranges)
// RUN: %refactor -find-rename-ranges -source-filename %s -pos="test" -is-function-like -old-name "subscript(x:)" >> %t.ranges/backticks-noncollapsible-separate-arglabel.swift.expected
// RUN: diff -u %S/Outputs/backticks-noncollapsible-separate-arglabel.swift.expected %t.ranges/backticks-noncollapsible-separate-arglabel.swift.expected
