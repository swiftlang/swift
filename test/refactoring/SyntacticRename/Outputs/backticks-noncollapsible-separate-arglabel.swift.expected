struct Foo {
    /*test:def*/<keywordBase>subscript</keywordBase>(<arglabel index=0>`x`</arglabel> <noncollapsibleparam index=0>y</noncollapsibleparam>: Int) -> Int { fatalError() }
}
Foo()/*test:ref*/[<callarg index=0>`x`</callarg><callcolon index=0>: </callcolon>10]

